Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift

```diff
@@ -51,6 +51,24 @@ public enum GermanKeyboardConstants {
     ["selectKeyboard", "ABC", "space", "ABC", "hideKeyboard"], // "undoArrow"
   ]
 
+  //added for expanded keyboard
+  static let letterKeysPadExpanded = [
+    ["^", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "ß", "'", "delete"],
+    ["indent", "q", "w", "e", "r", "t", "z", "u", "i", "o", "p", "ü", "+", "return"],
+    ["uppercase", "a", "s", "d", "f", "g", "h", "j", "k", "l", "ö", "ä", "#", "return"],
+    ["shift", "<", "y", "x", "c", "v", "b", "n", "m", ",", ".", "-", "shift"],
+    ["selectKeyboard", ".?123", "microphone", "space", ".?123", "scribble", "hideKeyboard"], // "undoArrow"
+  ]
+
+  //added for symbolKeysPadExpanded 
+  static let symbolKeysPadExpanded = [ 
+    ["'", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "<", ">", "delete"],
+    ["indent", "[", "]", "{", "}", "#", "%", "^", "*", "+", "=", "\"", "|", "return"], 
+    ["undo", "-", "/", ":", ";", "(", ")", "$", "&", "@", "£", "¥", "~", "return"], 
+    ["redo", " ", "...", ".", ",", "?", "!", "'", '"', "_", "€"], 
+    ["selectKeyboard", ".?123", "microphone", "space", ".?123", "scribble", "hideKeyboard"],
```

Comment:
See all of these are also experiencing formatting errors :) I can definitely fix this post merge if it's a problem 🙃